render/wlr_renderer.c

```diff
@@ -203,7 +203,7 @@ struct wlr_renderer *wlr_renderer_autocreate(struct wlr_egl *egl,
 
 	size_t all_config_attribs_len = config_attribs_len +
 		sizeof(gles2_config_attribs) / sizeof(gles2_config_attribs[0]);
-	EGLint all_config_attribs[all_config_attribs_len];
+	EGLint *all_config_attribs = malloc(sizeof(EGLint) * all_config_attribs_len);
 	if (config_attribs_len > 0) {
 		memcpy(all_config_attribs, config_attribs,
 			config_attribs_len * sizeof(EGLint));
```

Comment:
There's one return NULL in the out of context blob below as well

Reply:
Again, not sure what you're referring to because of this issue with out-of-view comments. Sorry!

Reply:
This leaks all_config_attribs:
```c
	if (!wlr_egl_init(egl, platform, remote_display, all_config_attribs,
			visual_id)) {
		wlr_log(WLR_ERROR, "Could not initialize EGL");
		return NULL;
	}
```